Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml

```diff
@@ -137,6 +137,13 @@
           - ItemMask
         restitution: 0.3
         friction: 0.2
+    ignition:
```

Comment:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/c3288711-fe3f-4fff-b485-286fdbb17978)

check fails is real

Reply:
it's added in the the PR linked in the description

Reply:
I'll probably just move the fix over to this PR and keep the firestack changes in the otther one

Reply:
ah, okay, I see. Then we wait.

Reply:
Should be fine to merge now